.circleci/config.yml

```diff
@@ -28,6 +28,25 @@ jobs:
         paths:
         - .build
 
+  generator:
+    docker:
+    - image: circleci/golang:1.10
+    working_directory: /go/src/github.com/prometheus/snmp_exporter/generator
+    resource_class: large
+
+    environment:
+      MIBDIRS: mibs:/usr/share/snmp/mibs
+    steps:
+    - checkout:
+        path: /go/src/github.com/prometheus/snmp_exporter
+    - run: sudo sh -c "echo 'deb http://deb.debian.org/debian stable non-free' > /etc/apt/sources.list.d/non-free.list"
+    - run: sudo apt-get update
+    - run: sudo apt-get -y install build-essential diffutils libsnmp-dev snmp-mibs-downloader
```

Comment:
What do we need snmp-mibs-downloader for? I'd expect the MIBs to be self-contained.